lib/flipper/spec/shared_adapter_specs.rb

```diff
@@ -205,7 +205,7 @@
 
     expect(subject.remove(feature)).to eq(true)
 
-    expect(subject.get(feature)).to eq(subject.default_config)
+    expect(subject.get(feature).values).to all(be_blank)
```

Comment:
This is because the value with this change can be:

```
{:actors=>#<Set: {}>, :boolean=>nil, :groups=>#<Set: {}>, :percentage_of_actors=>nil, :percentage_of_time=>nil}
```

And that is because if we delete, but a stale reader is reading old data, the stale reader might decide to write the old data. If we insert an empty hash on delete then we guarantee that can't happen.